pyproject.toml

```diff
@@ -16,11 +16,11 @@ numpy = ">=1.19.0"
 scipy = ">=1.6"
 formulaic = ">=1.0.0"
 lets-plot = ">=4.0.1"
-seaborn = ">=0.13.2"
```

Comment:
i think dropping this is breaking a handful of unit tests. no harm keeping it imo; esp since it also installs matplotlib automatically.